### PR TITLE
T1145 Private Keys

### PR DESCRIPTION
**Pull request by VakarisZ**
# Feature
> Reports SSH keys monkey managed to find
image

### CODE REVIEW

File: monkey/monkey_island/cc/ui/src/components/attack/techniques/Helpers.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import React from "react";
+
+export function RenderMachine(val){
+    return (
+      <span>{val.ip_addr} {(val.domain_name ? " (".concat(val.domain_name, ")") : "")}</span>
+    )
+}
+
+export function renderMachineFromSystemData(data) {
```
Comment (itaymmguardicore) [inline monkey/monkey_island/cc/ui/src/components/attack/techniques/Helpers.js]: briefly document what the function does (what it gets and returns)

Comment (itaymmguardicore) [inline monkey/monkey_island/cc/ui/src/components/attack/techniques/Helpers.js]: maybe rename data to machine

File: monkey/monkey_island/cc/services/attack/technique_reports/T1145.py [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+                         'data.ssh_info': {'$elemMatch': {'private_key': {'$exists': True}}}}},
+             {'$project': {'_id': 0,
+                           'machine': {'hostname': '$data.hostname', 'ips': '$data.network_info.networks'},
+                           'ssh_info': '$data.ssh_info'}}]
+
+    @staticmethod
+    def get_report_data():
+        ssh_info = list(mongo.db.telemetry.aggregate(T1145.query))
+
+        if ssh_info:
```
Comment (itaymmguardicore) [inline monkey/monkey_island/cc/services/attack/technique_reports/T1145.py]: you know what to do

File: monkey/monkey_island/cc/services/attack/technique_reports/T1145.py [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+
+class T1145(AttackTechnique):
+    tech_id = "T1145"
+    unscanned_msg = "Monkey didn't find any shh keys."
+    scanned_msg = ""
+    used_msg = "Monkey found ssh keys on machines in the network."
+
+    # Gets data about ssh keys found
+    query = [{'$match': {'telem_type': 'system_info_collection',
```
Comment (itaymmguardicore) [inline monkey/monkey_island/cc/services/attack/technique_reports/T1145.py]: telem_catagory

File: monkey/monkey_island/cc/ui/src/components/attack/techniques/Helpers.js (hunk 1 above)
Comment (VakarisZ) [reply monkey/monkey_island/cc/ui/src/components/attack/techniques/Helpers.js]: This function parses system info collector data to create a string representation for machine (same as render machine, just parses different format of data).

**Pull request merged**